mlrun/utils/notifications/notification/webhook.py

```diff
@@ -69,29 +69,31 @@ async def push(
             request_body["custom_html"] = custom_html
 
         if override_body:
-            list_edit_runs = []
-            for run in runs:
```

Comment:
its better to verify that runs is a list with `runs = runs or []` instead of calling a nested function for each value that has {{runs}} in it because that nested function parses the runs each time instead of doing it once.